stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/Constants.java

```diff
@@ -197,8 +197,9 @@
     public static final String SQL_FILE_DB_POSTFIX = "_FILE";
     public static final String SQL_MEMORY_DB_POSTFIX = "_MEMORY";
 
-    public static final String SQL_CONNECTOR_ERROR = "error";
-    public static final String SQL_EXCEPTION_OCCURED = "SQL Exception Occurred";
+    public static final String DATABASE_ERROR_RECORD_NAME = "DatabaseError";
+    public static final String DATABASE_ERROR_MESSAGE = "SQL Exception Occurred";
```

Comment:
I think we need to change the wording. Better use the word error than the exception. WDYT?